vpr/src/base/vpr_api.cpp

```diff
@@ -604,10 +611,41 @@ bool vpr_pack_flow(t_vpr_setup& vpr_setup, const t_arch& arch) {
             vpr_load_packing(vpr_setup, arch);
         } else {
             VTR_ASSERT(packer_opts.doPacking == STAGE_LOAD);
-            //Load a previous packing from the .net file
-            vpr_load_packing(vpr_setup, arch);
-            //Load cluster_constraints data structure here since loading pack file
-            load_cluster_constraints();
+
```

Comment:
This should be turned into a helper function that does the overall flat placement loading control flow (basically all this code); it keeps what is already a big overall control routine a bit higher-level and understandable.